src/utilities/get.ts

```diff
@@ -0,0 +1,17 @@
+import {GeneralObject} from '../types';
+
+function get(obj: GeneralObject | undefined, keypath: string) {
```

Comment:
Are we attempting to mimic lodash _.get method or use our own? The _.get function seems to take an array as a `keypath` as well as a string. Here's my hacky attempt at playing around with it:

 `function get(obj: GeneralObject | undefined, keypath: Array<any> | string) {
  if (obj == null) return undefined;
  let keys;
  if (typeof keypath === 'string') {
    keys = keypath.split('.');
  } else {
    keys = keypath;
  }`

Reply:
and a fun little use case from lodash:
` const object = {a: [{b: {c: 3}}]};`
   ` get(object, ['a', '0', 'b', 'c']);`
   ` // => 3`

Reply:
Create our own minimal function although I could add a form of this. I'll put something tomorrow